pkgs/top-level/perl-packages.nix

```diff
@@ -17207,13 +17207,19 @@ let
     };
   };
 
-  PkgConfig = buildPerlPackage {
+  PkgConfig = buildPerlPackage rec {
     pname = "PkgConfig";
     version = "0.25026";
     src = fetchurl {
       url = "mirror://cpan/authors/id/P/PL/PLICEASE/PkgConfig-0.25026.tar.gz";
       sha256 = "1862hzlkibqsgynrnwg43acycp4rlsv19gsybjwq39nnqb9mxfjd";
     };
+    # support cross-compilation by simplifying the way we get version during build
+    postPatch = lib.optionalString (stdenv.buildPlatform != stdenv.hostPlatform) ''
```

Comment:
Since this seems to be triggering rebuilds anyway, maybe we should do this unconditionally. (The substituteInPlace doesn't cause errors on failed match though, so maybe it doesn't matter.)

Reply:
Yeah, I agree - either way, we're not really tied to `$PkgConfig::VERSION` at this point, since we're already supplying `${version}` through the derivation.

As a CPAN maintainer, i is actually rather strange for me to see this `do { require "./lib/MyModule.pm"; $MyModule::VERSION }` construct in a `Makefile.PL` - nowadays it is usually the other way around (some authoring tool writes to the module via the canonical setting in the `Makefile.PL` :thinking: )

Reply:
Now updated, thanks for the suggestion! :pray: